addons/website/static/src/snippets/s_dynamic_snippet/000.js

```diff
@@ -122,6 +122,11 @@ const DynamicSnippet = publicWidget.Widget.extend({
                     'limit': parseInt(nodeData.numberOfRecords),
                     'search_domain': this._getSearchDomain(),
                     'with_sample': this.editableMode,
+                    // TODO adapt in master (see _bugfix_force_minimum_max_limit_to_16)
+                    // in python. The `forceMinimumMaxLimitTo16` value in the
+                    // dataset is there only in dynamic snippets whose options
+                    // have been configured after this fix was merged.
+                    '_bugfix_force_minimum_max_limit_to_16': !!nodeData.forceMinimumMaxLimitTo16,
```

Comment:
You can remove the whole diff in cont/main.py and go with:
```suggestion
                   'context': {
                        '_bugfix_force_minimum_max_limit_to_16': !!nodeData.forceMinimumMaxLimitTo16,
                    }
```
It will naturally go through your code in `website/models/website_snippet_filter.py` which will end up with this context key

Reply:
I tried that first, it did not work. But in fact it does if you put the `context` object inside the `params` one :shrug: I guess it's like that when calling route directly instead calling model methods :shrug:

Reply:
Yeah I was surprised too, I didn't put it in the param first when trying :grimacing: